### PR TITLE
Bank-Packet

### PR DESCRIPTION
**Pull request by sctnightcore**

### CODE REVIEW

File: src/Network/Receive/ServerType0.pm [hunk 1]
```diff
@@ -570,6 +570,9 @@ sub new {
 		'099D' => ['received_characters', 'v a*', [qw(len charInfo)]],
 		'099F' => ['area_spell_multiple2', 'v a*', [qw(len spellInfo)]], # -1
 		'09A0' => ['sync_received_characters', 'V', [qw(sync_Count)]],
+		'09A6' => ['banking_status', 'V2 x2',[qw(inbank reason)]], #rA : reason = 0
+		'09A8' => ['banking_status_deposit', 'v V3',[qw(reason inbank onhand)]],
```
Comment (alisonrag) [inline src/Network/Receive/ServerType0.pm]: have more unpack string than vars to receive

File: src/Network/Send.pm [hunk 2]
```diff
@@ -1750,4 +1750,28 @@ sub sendAdoptReply {
 	debug "Sent Adoption Reply.\n", "sendPacket", 2;
 }
 
+sub sendbankcheck {
```
Comment (alisonrag) [inline src/Network/Send.pm]: missing the camel case use

File: src/Network/Receive/ServerType0.pm (hunk 1 above)
Comment (moneyisgod) [reply src/Network/Receive/ServerType0.pm]: 09A8 <Reason>W <Money>Q <balance>L (PACKET_ZC_ACK_BANKING_DEPOSIT)

WORD, QWORD, LONG

Comment (sctnightcore) [reply src/Network/Receive/ServerType0.pm]: FIXED :  c85ed1e

File: src/Network/Send.pm (hunk 2 above)
Comment (sctnightcore) [reply src/Network/Send.pm]: Fixed : e3a0714

**Comment by alisonrag** [comment]
@Nipodemos 
now this pull is confliting with master

**Comment by Nipodemos** [comment]
@sctnightcore can you redo this Pull Request?

**Comment by lututui** [comment]
I've rebased this but there are still some stuff that needs to be addressed - and by that I mean the unpack string is clearly wrong

**Comment by sctnightcore** [comment]
I have copied from my old PR.
Tested successfully.

**Comment by Fadreus** [comment]
iRO don't have it.

**Comment by AlexandreVitorino** [comment]
> 
> 
> iRO don't have it.

Oh man amazing, it's almost than 5am and we are awake! More power to you bro!

**Comment by alisonrag** [comment]
@AlexandreVitorino
Sakexe_0 will always base private servers based in kRO ragexeRE
Servertype0 will always base official servers
We are cleaning, but we not merged the mentioneds servertypes

**Pull request closed without merging**